x-pack/plugins/synthetics/common/runtime_types/synthetics_private_locations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import * as t from 'io-ts';
+
+export const PrivateLocationType = t.type({
```

Comment:
Why is this all on its own in it's own file? Wondering if we should include it in the generic locations file in monitor_management